README.md

```
@@ -31,6 +474,49 @@ In order to allow the app to use modern SwiftUI features, it supports the follow
+| Typing    | `<roomId>::$chat::$typingIndicators` |
+
+---
+
 ## Contributing
 
-For information on how to contribute to this repository, please see the [contributing guidelines](CONTRIBUTING.md).
+For guidance on how to contribute to this project, see the [contributing guidelines](CONTRIBUTING.md).
+
+## Support, feedback and troubleshooting
+
+Please visit http://support.ably.com/ for access to our knowledge base and to ask for any assistance. You can also view
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)